.gitignore

```diff
@@ -44,3 +44,6 @@ yarn-error.log*
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+#enviroments variables
+.env
```

Comment:
could you add a file `.env.example` where you explain the variables that are suppose to go in `.env`, please?